[CI] IndexCommitTimestampFieldRangeTests testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange failing

**Build Scans:**
- [elasticsearch-intake #39589 / part2](https://gradle-enterprise.elastic.co/s/sntb5ofrq55ay)
- [elasticsearch-pull-request #145187 / part-2](https://gradle-enterprise.elastic.co/s/k6r5vnigcnyjs)
- [elasticsearch-pull-request #145182 / part-2](https://gradle-enterprise.elastic.co/s/zamqib67sj2rk)
- [elasticsearch-pull-request #145187 / part-2](https://gradle-enterprise.elastic.co/s/tg5gwzkpbx2f4)
- [elasticsearch-pull-request #145176 / part-2](https://gradle-enterprise.elastic.co/s/bkvgsvlrxehz2)
- [elasticsearch-pull-request #145182 / part-2](https://gradle-enterprise.elastic.co/s/wvbpgjrgjxsr4)
- [elasticsearch-pull-request #145176 / part-2](https://gradle-enterprise.elastic.co/s/4337noot56e4a)
- [elasticsearch-intake #39583 / part2](https://gradle-enterprise.elastic.co/s/lckllgakjjlaa)
- [elasticsearch-pull-request #145154 / part-2](https://gradle-enterprise.elastic.co/s/fsswdtgab35l2)
- [elasticsearch-pull-request #145154 / part-2](https://gradle-enterprise.elastic.co/s/kp2xcavm34hdk)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:stateless:test" --tests "org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests.testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange" -Dtests.seed=9BA8A8DA8105B4EC -Dtests.locale=oc-ES -Dtests.timezone=Asia/Kathmandu -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange
- [main] 1 consecutive failures in step part2
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 20 failures in test testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange (6.1% fail rate in 329 executions)
- [main] 3 failures in step part2 (4.3% fail rate in 70 executions)
- [main] 17 failures in step part-2 (8.1% fail rate in 209 executions)
- [main] 3 failures in pipeline elasticsearch-intake (4.3% fail rate in 70 executions)
- [main] 9 failures in pipeline elasticsearch-pull-request (4.4% fail rate in 204 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -498,9 +498,6 @@ tests:
   method: testProcessOnceOnPrimary
   issue: https://github.com/elastic/elasticsearch/issues/148943
 - class: org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests
-  method: testSoftDeletesAreAlmostAlwaysDisregardedForTimestampRange
-  issue: https://github.com/elastic/elasticsearch/issues/148966
-- class: org.elasticsearch.xpack.stateless.commits.IndexCommitTimestampFieldRangeTests
   method: testFieldValueRangeForBatchedCompoundCommit
   issue: https://github.com/elastic/elasticsearch/issues/148961
 - class: org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT

